src/sentry/preprod/models.py

```diff
@@ -116,6 +116,8 @@ def as_choices(cls):
     # Size analysis wont necessarily be run on every artifact (based on quotas)
     analysis_file_id = BoundedBigIntegerField(db_index=True, null=True)
 
+    install_file_id = BoundedBigIntegerField(db_index=True, null=True)
```

Comment:
nit: can you add a better name for this like `installable_{app|artifact}_file_id`? a comment would also be helpful.